metaflow/plugins/aip/argo_utils.py

```diff
@@ -12,155 +12,290 @@
 logger = _get_aip_logger()
 
 
-def run_argo_workflow(
-    kubernetes_namespace: str,
-    template_name: str,
-    parameters: Optional[dict] = None,
-    wait_timeout: Union[int, float, datetime.timedelta] = 0,
-    **kwarg,  # Other parameters for wait function
-) -> Tuple[str, str]:
-    try:
-        # TODO(talebz): add tag of origin-run-id to correlate parent flow
-        workflow_manifest: Dict[str, Any] = ArgoClient(
-            namespace=kubernetes_namespace,
-        ).trigger_workflow_template(template_name, parameters)
-    except Exception as e:
-        raise AIPException(str(e))
-
-    argo_run_id = workflow_manifest["metadata"]["name"]
-    argo_run_uid = workflow_manifest["metadata"]["uid"]
-
-    if wait_timeout:  # int, float and datetime.timedelta all evaluates to False when 0
-        wait_for_argo_run_completion(
-            run_id=argo_run_id,
-            kubernetes_namespace=kubernetes_namespace,
-            wait_timeout=wait_timeout,
-            **kwarg,
-        )
+class ArgoHelper:
+    def __init__(self, kubernetes_namespace: str):
```

Comment:
can this namespace default to `from metaflow.metaflow_config import KUBERNETES_NAMESPACE` ?